src/version.ts

```diff
@@ -1 +1 @@
-export const LIB_VERSION = "3.0.4";
+export const LIB_VERSION = "3.0.4"
```

Comment:
there's a ton of reasons why semicolon usage should be enforced; can that be enabled in the prettier config? fine to do in a followup too

Reply:
I noticed that too. Will enable in a separate PR